app/classes/battleMethods.py

```diff
@@ -0,0 +1,26 @@
+from app.classes.knight import Knight
+
+
+def fight(k_1: Knight, k_2: Knight) -> None:
```

Comment:
Use more descriptive variable names instead of `k_1 `and `k_2`. Same relates to the `results` function as well